src/Compilers/CSharp/Portable/FlowAnalysis/NullableWalker.cs

```diff
@@ -115,33 +115,119 @@ public static void Analyze(CSharpCompilation compilation, Symbol member, BoundNo
 
         protected override void Normalize(ref LocalState state)
         {
+            int oldNext = state.Capacity;
             state.EnsureCapacity(nextVariableSlot);
+            for (int slot = oldNext; slot < nextVariableSlot; slot++)
+            {
+                var value = !GetDefaultState(ref state, slot);
+                state[slot] = value;
+            }
+        }
+
+        private bool? GetDefaultState(ref LocalState state, int slot)
```

Comment:
>GetDefaultState [](start = 22, length = 15)

It feels confusing that the method is called GetDefaultState, but its return value should be negated before it can be used as a state. Consider returning ready to use values.

Reply:
Done.